src/controllers/auth-controller.ts

```diff
@@ -22,6 +23,7 @@ export default class AuthController extends AbstractController {
     this.router.post('/logout', this.logout);
     this.router.post('/login', this.login);
     this.router.post('/refresh', this.refresh);
+    this.router.post('/changePassword', authRequired, this.changePassword);
```

Comment:
camelCase variant not so good variant as url

Reply:
'/change-password' was applied